template/Makefile.j2

```diff
@@ -15,6 +15,9 @@ VERSION := $(shell cargo metadata --format-version 1 | jq '.packages[] | select(
 ## Docker related targets
 docker-build:
 	docker build --force-rm -t "docker.stackable.tech/stackable/{[ operator.product_string }]-operator:${VERSION}" -f docker/Dockerfile .
+{[% if operator.product_string in ['opa'] %}]
+	docker build --force-rm -t "docker.stackable.tech/stackable/{[ operator.product_string }]-bundle-builder:${VERSION}" -f docker/Dockerfile.builder .
+{[% endif %}]
 
 docker-build-latest: docker-build
 	docker tag "docker.stackable.tech/stackable/{[ operator.product_string }]-operator:${VERSION}" \
```

Comment:
Does this also need changing for the bundle builder?

Reply:
We can it it's not needed. The image for the builder is never installed from `latest` but always from the exact same version as the operator.
To put it differently: the builder is image is a "product image" that always comes with the same version as the operator.

Reply:
ok